spec/controllers/communicarts_controller_spec.rb

```diff
@@ -130,24 +130,6 @@
 
 
     context 'Request token' do
-      it 'fails when the token does not exist' do
-        approval_params_with_token[:cch] = nil
-        bypass_rescue
-        expect { put 'approval_response', approval_params_with_token }.to raise_error(AuthenticationError)
-      end
-
-      it 'fails when the token has expired' do
-        token.update_attributes(expires_at: 8.days.ago)
-        bypass_rescue
-        expect { put 'approval_response', approval_params_with_token }.to raise_error(AuthenticationError)
-      end
-
-      it 'fails when the token has already been used once' do
-        token.update_attributes(used_at: 1.hour.ago)
-        bypass_rescue
-        expect { put 'approval_response', approval_params_with_token }.to raise_error(AuthenticationError)
-      end
```

Comment:
I think it still might be useful to have a smoke test at the controller/request level, even if it just uses [mocks](https://relishapp.com/rspec/rspec-mocks/docs) to ensure that the appropriate policy is being called.